stingray/crossspectrum.py

```diff
@@ -540,6 +540,71 @@ def time_lag(self):
         else:
             raise AttributeError("Object has no attribute named 'time_lag' !")
 
+    def plot(self, labels=None, axis=None, title=None, marker='-', save=False,
+             filename=None):
+        """
+        Plot the amplitude of the cross spectrum vs. the frequency using ``matplotlib``.
+
+        Parameters
+        ----------
+        labels : iterable, default ``None``
+            A list of tuple with ``xlabel`` and ``ylabel`` as strings.
+
+        axis : list, tuple, string, default ``None``
+            Parameter to set axis properties of the ``matplotlib`` figure. For example
+            it can be a list like ``[xmin, xmax, ymin, ymax]`` or any other
+            acceptable argument for the``matplotlib.pyplot.axis()`` method.
+
+        title : str, default ``None``
+            The title of the plot.
+
+        marker : str, default '-'
+            Line style and color of the plot. Line styles and colors are
+            combined in a single format string, as in ``'bo'`` for blue
+            circles. See ``matplotlib.pyplot.plot`` for more options.
+
+        save : boolean, optional, default ``False``
+            If ``True``, save the figure with specified filename.
+
+        filename : str
+            File name of the image to save. Depends on the boolean ``save``.
+        """
+        try:
+            import matplotlib.pyplot as plt
+        except ImportError:
+            raise ImportError("Matplotlib required for plot()")
+
+        fig = plt.figure()
+        fig = plt.plot(self.freq, np.abs(self.power), marker)
```

Comment:
How about plotting also the real and imaginary parts? Also, can you show that this works, at least as a picture in a comment?

Reply:
(I ask about the real and imaginary parts because the real part - or _cospectrum_ - is used as a proxy for the power spectrum in some applications)

Reply:
I just took `Lightcurve` plot test and used it here. It looks like this:
```
    def test_plot_simple(self):
        lc = Lightcurve(self.times, self.counts)
        lc.plot()
        assert plt.fignum_exists(1)
```

Reply:
@matteobachetti Are you suggesting to plot Argand Diagram? Do want it to be a separate function?

Reply:
No, just the real part and the imaginary part. Something like
```
plt.plot(self.freq, np.abs(self.power), color=color1)
plt.plot(self.freq, self.power.real, color=color2, alpha=0.5)
plt.plot(self.freq, self.power.imag, color=color3, alpha=0.5)
```
or anything prettier :)